.github/kubernetes-versions.json

```diff
@@ -0,0 +1,5 @@
+[
+    "v1.25.11+k3s1",
+    "v1.26.6+k3s1",
+    "v1.27.3+k3s1"
```

Comment:
These should match what the repository provides: https://github.com/kairos-io/packages/blob/main/packages/k8s/k3s/collection.yaml

I wonder if we should parse the `collection.yaml` from the remote location directly :thinking: . This way, when we want to change the versions we support, we will only have to change the versions built in the packages repository and this pipeline will pick it up automatically.

Reply:
I would say yes, otherwise we will need to do manual stuff no?

Reply:
Done (I think, we will know when the pipeline runs)

Reply:
https://github.com/kairos-io/kairos/pull/1645/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR40-R42